bpf/bpf_host.c

```diff
@@ -281,7 +281,8 @@ handle_ipv6(struct __ctx_buff *ctx, __u32 secctx, const bool from_host)
 			return CTX_ACT_OK;
 
 		return ipv6_local_delivery(ctx, l3_off, secctx, ep,
-					   METRIC_INGRESS, from_host);
```

Comment:
Given we're talking about delivery to the local pod here, I would expect it to always be "ingress".